images/ci-openshift-base.rhel9.yml

```diff
@@ -13,7 +13,15 @@ content:
         enabled: false
       mirror: true
       mirror_manifest_list: true
-      upstream_image: registry.ci.openshift.org/ocp/builder:rhel-9-enterprise-base-multi-openshift-{MAJOR}.{MINOR}
+      upstream_image: registry.ci.openshift.org/ocp/{MAJOR}.{MINOR}:base-rhel9
+
+    okd_alignment:
```

Comment:
Ideally we could produce an image based on https://quay.io/centos/centos:stream9 with our https://github.com/openshift-eng/ocp-build-data/blob/openshift-4.18/ci_images/dnf_wrapper.sh script installed on top of it.
For OKD, this base image can't be based on RHEL. It must be based on centos stream. This makes it impossible to build the desired image in OSBS.
okd_alignment only instructs the creation of PRs in openshift/release which build OKD variants of images when PR merge in upstream component repositories.